package-lock.json

```diff
@@ -20151,19 +20151,19 @@
       "dev": true,
       "requires": {
         "chalk": "2.4.2",
-        "conventional-changelog": "3.1.10",
```

Comment:
This package-lock shouldn't be modified. If there are dep updates, we typically handle these in a separate PR.

Reply:
got it, those changes have been removed